apps/projects/app/components/Panel/NewProject/NewProject.js

```diff
@@ -63,160 +206,127 @@ const NewProject = () => {
-
-                    <Button
-                      mode="strong"
-                      wide
-                      onClick={handleNewProject}
-                      disabled={repoSelected < 0}
-                    >
-                      Submit
-                    </Button>
-                  </div>
-                )
+const createProject = () => {}
+
+const NewProject = ({ handleGithubSignIn }) => {
+  const [ title, setTitle ] = useState('')
+  const [ description, setDescription ] = useState('')
+  const theme = useTheme()
+  const { appState: { github = { status: STATUS.INITIAL } } } = useAragonApi()
+
+  return (
+    <PanelContent>
+      {
+        disableDecoupledProjects ? (
+          <GitHubRepoList handleGithubSignIn={handleGithubSignIn} />
+        ) : (
+          <React.Fragment>
+            <InfoBox>
+              Create a new project that belongs and operates entirely within this application, or synchronize one from GitHub.
+            </InfoBox>
+
+            <FormField
+              label="Title"
+              required={github.status !== 'authenticated' || title !== ''}
+              input={
+                <TextInput
+                  wide
+                  value={title}
+                  onChange={e => setTitle(e.target.value)}
+                  aria-label="Title"
+                />
               }
+            />
 
-              if (loading) return (
-                <RepoInfo>
-                  <LoadingAnimation />
-                  <div>Loading repositories...</div>
-                </RepoInfo>
-              )
-
-              if (error) return (
-                <RepoInfo>
-                  <Text size="xsmall" style={{ margin: '20px 0' }}>
-                      Error {JSON.stringify(error)}
-                  </Text>
-                  <Button wide mode="strong" onClick={() => refetch()}>
-                      Try refetching?
-                  </Button>
-                </RepoInfo>
-              )
-            }}
-          </Query>
-        </div>
-      </div>
-    </React.Fragment>
+            {title !== '' ? (
+              <>
+              <FormField
+                label="Description"
+                input={
+                  <TextInput.Multiline
+                    name="description"
+                    rows="3"
+                    onChange={e => setDescription(e.target.value)}
+                    value={description}
+                    wide
+                    aria-label="Description"
+                  />
+                }
+              />
+              <Button
+                mode="strong"
+                wide
+                onClick={createProject}
+                disabled={title === ''}
+              >
+                Submit
+              </Button>
+              </>
+            ) : (
+              <>
+              <ThematicBreak
+                color={`${theme.surfaceContentSecondary}`}
+                linecolor={`${theme.border}`}
```

Comment:
I think it would clean up this component nicely if these props didn't have to be passed in

```jsx
const ThematicBreak = ({ children }) => {
  const theme = useTheme()
  return (
    <div css={`
      ...
    `}>
      {children}
    </div>
  )
}
```

Or maybe if the "or" is also included in the component itself:

```jsx
const ThematicBreak = () => {
  const theme = useTheme()
  return (
    <div css={`
      ...
    `}>
      or
    </div>
  )
}
```

Reply:
Ok. The scecond one makes sense - this component is only used once, it will probably retain colors and text if it needs to be reused, and it can alway be re-customized later.. Changed.